x/x.go

```diff
@@ -146,6 +146,7 @@ const (
 	AccessControlAllowedHeaders = "X-Dgraph-AccessToken, " +
 		"Content-Type, Content-Length, Accept-Encoding, Cache-Control, " +
 		"X-CSRF-Token, X-Auth-Token, X-Requested-With"
+	DgraphCostHeader = "numUids"
```

Comment:
Oh sorry, I should have been explicit 🙈.. Let's keep it similar to graphql.

Dgraph-TouchedUids